resources/build/trigger-builds.inc.sh

```diff
@@ -69,30 +69,31 @@ function triggerGitHubActionsBuild() {
   local IS_TEST_BUILD="$1"
   local GITHUB_ACTION="$2"
   local GIT_BRANCH="${3:-master}"
-  local GIT_REF GIT_SHA
+  local GIT_REF GIT_HEAD_REF
 
-  local GITHUB_SERVER=https://api.github.com/repos/keymanapp/keyman/dispatches
+  local GITHUB_SERVER=https://api.github.com/repos/keymanapp/keyman
 
   if [ "${action:-""}" == "commit" ]; then
     # This will only be true if we created and pushed a tag
     GIT_REF="refs/tags/release@$VERSION_WITH_TAG"
-    GIT_SHA="$(git rev-parse "${GIT_REF}")"
+    GIT_HEAD_REF="${GIT_REF}"
     GIT_EVENT_TYPE="${GITHUB_ACTION}: release@${VERSION_WITH_TAG}"
   elif [[ $GIT_BRANCH != stable-* ]] && [[ $GIT_BRANCH =~ [0-9]+ ]]; then
     GIT_REF="refs/pull/${GIT_BRANCH}/merge"
-    GIT_SHA="$(git rev-parse "refs/pull/${GIT_BRANCH}/head")"
+    GIT_HEAD_REF="refs/pull/${GIT_BRANCH}/head"
     GIT_EVENT_TYPE="${GITHUB_ACTION}: PR #${GIT_BRANCH}"
     GIT_BRANCH="PR-${GIT_BRANCH}"
   else
     GIT_REF="refs/heads/${GIT_BRANCH}"
-    GIT_SHA="$(git rev-parse "${GIT_REF}")"
+    GIT_HEAD_REF="${GIT_REF}"
     GIT_EVENT_TYPE="${GITHUB_ACTION}: ${GIT_BRANCH}"
   fi
 
-  local DATA="{\"event_type\": \"$GIT_EVENT_TYPE\", \
-      \"client_payload\": { \
+  local DATA="{
+    \"ref\": \"$GIT_REF\", \
+    \"inputs\": { \
         \"ref\": \"$GIT_REF\", \
-        \"sha\": \"$GIT_SHA\", \
+        \"headRef\": \"$GIT_HEAD_REF\", \
         \"branch\": \"$GIT_BRANCH\", \
         \"isTestBuild\": \"$IS_TEST_BUILD\" \
     }}"
```

Comment:
I'm not all that comfortable with this change because this means that we are no longer building a specific commit from the trigger. I was deliberately targeting a specific commit in our existing model.

Reply:
Turns out we don't need this variable at all - it's removed in one of the following PRs. 

We're still building a specific commit - see `ref` at the top of `DATA`.